model-engine/tests/unit/domain/test_llm_use_cases.py

```diff
@@ -953,3 +958,53 @@ async def test_delete_public_inference_model_raises_not_authorized(
         await use_case.execute(
             user=user, model_endpoint_name=llm_model_endpoint_sync[0].record.name
         )
+
+
+@pytest.mark.parametrize(
+    "framework",
+    [
+        LLMInferenceFramework.DEEPSPEED,
+        LLMInferenceFramework.LIGHTLLM,
+        LLMInferenceFramework.TEXT_GENERATION_INFERENCE,
+        LLMInferenceFramework.VLLM,
+    ],
+)
+@pytest.mark.parametrize("image_tag", ["0.9.3-launch_s3"])
+@pytest.mark.parametrize(
+    "checkpoint_path",
+    [
+        "s3://scale-ml/testing/llama-2-13b-bin-path/",
```

Comment:
Are we actually accessing this bucket every time we run this unit test? If so, I think that means we need to run with appropriate AWS permissions, maybe we should create a fake s3 bucket object for bin-path and safetensors-path so that we don't need to do this.

Reply:
@sam-scale okay we have to remove this for sure because we can't mention our s3 bucket location in the public repo.